streams/za.m3u

```diff
@@ -30,28 +30,22 @@ https://webstreaming-8.viewmedia.tv/web_119/Stream/playlist.m3u8
 #EXTINF:-1 tvg-id="ROVTV.za",ROV TV (576p)
 https://webstreaming.viewmedia.tv/web_012/Stream/playlist.m3u8
 #EXTINF:-1 tvg-id="SABC1.za",SABC 1 (720p)
-http://185.59.221.131:8081/live/sabc1/playlist.m3u8
+https://sabconeta.cdn.mangomolo.com/sabc1/smil:sabc1.stream.smil/chunklist_b1600000_t64NzIwcA==.m3u8
```

Comment:
Geo-blocked, please show if it's working for you in VLC

Reply:
![Screenshot_20240830_225417](https://github.com/user-attachments/assets/3219ac64-c464-4ff6-a7aa-407b713649ac)
![Screenshot_20240830_225446](https://github.com/user-attachments/assets/77e5f36d-4e11-4324-9bb9-d75b78ec013f)